### ISSUE TITLE
Ref has not yet been attached to a dialog node when attempting to call `createAriaHider`

### ISSUE THREAD

**Issue by PaulRBerg** [reach/reach-ui#336]
### Current Behavior

I'm using DialogContent and the following warning gets printed in the console:

```
A ref has not yet been attached to a dialog node when attempting to call `createAriaHider`
```

### Expected behavior

There should be no warning.

### Reproducible example

[Here's a CodeSandbox](https://codesandbox.io/s/objective-bash-e8970).

### Your environment

| Software                 | Name/Version(s) |
| ------------------------ | --------------- |
| @reach/dialog | 0.4.0 |
| React | 16.8.6 |
| Browser | Brave V0.69.135 Chromium: 77.0.3865.120 |
| npm/yarn                 | 1.19.1 |
| Operating System         | macOS |

**Comment by chancestrickland** [comment]
Fixed in 0.5.0. Feel free to test by bumping the dependency in your repro @PaulRBerg

**Comment by PaulRBerg** [comment]
Yup, all good now! Thanks.
